api/kubernetes/kube-prometheus-stack-template.yaml

```diff
@@ -0,0 +1,73 @@
+namespace: $MONITORING_NAMESPACE_VAR
```

Comment:
We need here to add the nodeSelector for all prometheus pods aligned with new kubernetes node pool infra.
![image](https://user-images.githubusercontent.com/558364/188594243-7091ffa2-c993-4701-a94b-2cd966463ad9.png)